src/GPU3D.h

```diff
@@ -80,6 +81,58 @@ struct Polygon
 
 };
 
+struct FrameDump
```

Comment:
Are you _only_ dumping the 3D renderer's state? Or do you intend to include the 2D renderers as well?

Reply:
Currently just 3d renderer, i do intend to leave room for the 2d renderer to be included in the future, but as of now, no